### PR TITLE
Document actions

### PR DESCRIPTION
**Pull request by s-atama**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -90,6 +90,11 @@ async function getConfig() {
               label: 'Composer Studio',
               position: 'left'
             },
+            {
+              to: '/docs/references',
+              label: 'References',
+              position: 'right'
+            },
```
Comment (paul-at-atama) [inline docusaurus.config.js]: Based on what I've seen in other [docs sites](https://docusaurus.io/showcase), I think it should be "Reference" (singular)

**Review by paul-at-atama — COMMENTED** [review]
You probably already saw that the build fails because there are more links to update across the board.

**Review by paul-at-atama — COMMENTED** [review]
Couple more thoughts on the structure:
1. Should the guides in Core still have next.js and hydrogen in it? If those are related to experiences, would they make more sense under Studio?
2. Depending on the answer #1, do we need a sub structure under Composer Core with "developer-guides" (next.js) and "user-guide" (bc, bcb)?

**Comment by s-atama** [comment]
* I fixed all links so docusaurus is happily building
* I moved the next.js and hydrogen guides to Composer Studio - I missed them before.

**Review by paul-at-atama — APPROVED** [review]

**Pull request merged**